usage/app/model/UsageTable.scala

```diff
@@ -35,6 +35,10 @@ class UsageTable(config: UsageConfig) extends DynamoDB(config, config.usageRecor
   }
 
   def queryByImageId(id: String): Future[Set[MediaUsage]] = Future {
+
+    if (id.isEmpty)
```

Comment:
Is it worth doing a `.trim` here, on the off chance that `"  "` is passed in as the ID?

Reply:
Looking back, this is a heroic spot @jennygrahamjones! 👍